packages/openid4vc/src/openid4vc-issuer/OpenId4VcIssuerApi.ts

```diff
@@ -30,7 +30,7 @@ export class OpenId4VcIssuerApi {
   }
 
   public async getByIssuerId(issuerId: string) {
-    return this.openId4VcIssuerService.getByIssuerId(this.agentContext, issuerId)
+    return this.openId4VcIssuerService.getIssuerByIssuerId(this.agentContext, issuerId)
```

Comment:
Nit, but `getIssuerById` makes more sense to me

Reply:
Yess it WOULD make more sense, BUT... the issuerRecord has an id and issuerId property. The reason is that the issuerId is used publicly (in the issuer url), while the record id is internal.

But maybe it should'nt be called issuerId then (but something like publicId)?